senders/pagerduty/send.go

```diff
@@ -17,7 +18,7 @@ const summaryMaxChars = 1024
 // SendEvents implements Sender interface Send
 func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.ContactData, trigger moira.TriggerData, plots [][]byte, throttled bool) error {
 	event := sender.buildEvent(events, contact, trigger, plots, throttled)
-	_, err := pagerduty.ManageEvent(event)
+	_, err := pagerduty.ManageEventWithContext(context.TODO(), event)
```

Comment:
I think we shouldn't merge `context.TODO()` in master.

Let's use `context.Backgound()` here for the time being

Reply:
Fixed